norma/urls.py

```diff
@@ -1,23 +1,22 @@
 from django.conf.urls import include, url
 
-from norma.views import (NormaEditView, NormaIncluirView, NormaPesquisaView,
-                         NormaTaView, PesquisaNormaListView,
-                         assunto_norma_crud, norma_temporario_crud,
-                         tipo_norma_crud)
+from norma.views import (AssuntoNormaCrud, NormaEditView, NormaIncluirView,
+                         NormaPesquisaView, NormaTaView, NormaTemporarioCrud,
+                         PesquisaNormaListView, TipoNormaCrud)
 
-# norma_url_patterns = norma_crud.urlpatterns + []
-norma_url_patterns = norma_temporario_crud.urlpatterns + [
+# FIXME???? usar NormaCrud ????
```

Comment:
esse código só está assim de forma temporária, criado no início do projeto para apenas dar uma tela básica de listagem de normas para a app compilação... a implementação da app norma é independente e não sei em que estágio está... para a compilação é relevante apenas que se mantenha o código abaixo:

`url(r'^norma/(?P<pk>[0-9]+)/ta$', NormaTaView.as_view(), name='ta')`

bem como a classe `NormaTaView` que está em `norma.views`